aiographite/aiographite.py

```diff
@@ -0,0 +1,156 @@
+from .graphite_encoder import GraphiteEncoder
+import asyncio
+import socket
+import time
+from aiographite.protocol import PlaintextProtocol, PickleProtocol
+from typing import Tuple, List, Callable
+
+
+class AioGraphiteSendException(Exception):
+    pass
+
+
+class AIOGraphite:
+
+    def __init__(self, graphite_server, graphite_port, protocol, loop=None):
```

Comment:
having a default port here is cool. in fact, we can probably stick the default port in the protocol, and use that as a default.